src/wp-content/themes/twentynineteen/inc/back-compat.php

```diff
@@ -37,7 +37,7 @@ function twentynineteen_switch_theme() {
  */
 function twentynineteen_upgrade_notice() {
 	printf(
-		'<div class="error"><p>%s</p></div>',
+		'<div class="notice notice-error"><p>%s</p></div>',
```

Comment:
Don't change back-compat classes, as the context where these run may not include the necessary support.